query/src/sql/planner/binder/mod.rs

```diff
@@ -18,9 +18,11 @@ pub use aggregate::AggregateInfo;
 pub use bind_context::BindContext;
 pub use bind_context::ColumnBinding;
 use common_ast::ast::Statement;
+use common_ast::ast::TimeTravelPoint;
 use common_datavalues::DataTypeImpl;
 use common_exception::ErrorCode;
 use common_exception::Result;
+use sqlparser::ast::Instant;
```

Comment:
Maybe it's better to introduce a internal `Instant` struct.

And is it possible to use an exists struct instead?

Reply:
`sqlparser::ast::Instant` is an new struct that we introduced to `sqlparser-rs`

https://github.com/datafuse-extras/sqlparser-rs/blob/7d74a3c5fc78f8d0c0f7e4e7e14d7058f94d8841/src/ast/query.rs#L321-L325

Shall we introduce another intermediate data structure that  `TimeTravelPoint` and `sqlparser::ast::Instant` can both convert into?

Reply:
It's good to have one. We will deprecate the sqlparser crate eventually.

Reply:
OK

Reply:
pls have look at commit `Introduce NavigationPoint` [#5546572a36e36e03473f945fc31b555374c957dc](https://github.com/datafuselabs/databend/pull/5628/commits/5546572a36e36e03473f945fc31b555374c957dc)